dev/source/docs/code-overview-adding-a-new-parameter.rst

```diff
@@ -127,6 +127,8 @@ characters.
 You're done!  The new parameter can be access from the main code (not
 the libraries) as g.my_new_parameter.
 
+.. note:: As the ardupilot is improving, the global parameter class is filling up. To avoid getting past the maximum number of parameters in Parameter class, another class for global paramters (named 'ParametersG2') has been introduced. Adding parameters to this class is similar to adding paramters in libraries. The paramters added to this class can also be accessed from the main code.
```

Comment:
```suggestion
.. note:: ArduPilot vehicle's global parameter classes are crowded. To avoid exceeding the maximum number of parameters in a vehicle's Parameter class, another class for global parameters (named 'ParametersG2') has been introduced. Adding parameters to this class is similar to adding parameters in libraries. The parameters added to this class can similarly be accessed from within the vehicle's main code.
```

Reply:
Thanks! I've pushed the changes. Please check.